db/seeds/test_data.rb

```diff
@@ -501,5 +501,6 @@
 end
 
 ["Capita", "Teach First", "UCL Institute of Education", "Best Practice Network", "Ambition Institute", "Education Development Trust"].each do |provider|
-  ValidTestDataGenerator::LeadProviderPopulater.delay.call(name: provider, total_schools: 10, participants_per_school: 10)
+  ValidTestDataGenerator::LeadProviderPopulater.delay.call(name: provider, total_schools: 1, participants_per_school: 10)
+  ValidTestDataGenerator::NPQLeadProviderPopulater.delay.call(name: provider, total_schools: 1, participants_per_school: 10)
```

Comment:
Surely we should only do this for NPQ lead providers?

Reply:
Ah, good spot. And we should do it for all of them probably.